src/main/java/at/hannibal2/skyhanni/config/ConfigManager.kt

```diff
@@ -62,6 +66,7 @@ class ConfigManager {
     private val jsonHolder: Map<ConfigFileType, Any> = EnumMap(ConfigFileType::class.java)
 
     lateinit var processor: MoulConfigProcessor<Features>
+    private var hasBeenInited = false
```

Comment:
is this variable meant to be "hasBeenInitiated"?

Reply:
fixed